specification/appendix/commitment_discounts/commitment_flexibility.md

```diff
@@ -1,8 +1,10 @@
-# Size-Flexibility
+# Commitment Flexibility
 
-A usage-based commitment discount obligates a customer to a usage amount for one or more related SKUs in return for reduced rates.  For example, a usage-based commitment discount is purchased to cover a specific database SKU, this commitment will cover every hour over the term where at least one instance of this SKU is running. It can also cover eligible resources wholly, where 1 hour is covered by 1 resource, or partially, where 1 committed hour can be split across multiple resources or 1 resource can be fully covered with unused commitment discount costs. These latter two examples are commonly known as *size-flexibility*.
+A usage-based commitment discount obligates a customer to a usage amount for one or more related SKUs in return for reduced rates.  For example, a usage-based commitment discount is purchased to cover a specific database SKU, this commitment will cover every hour over the term where at least one instance of this SKU is running. It can also cover eligible resources wholly, where 1 hour is covered by 1 resource, or partially, where 1 committed hour can be split across multiple resources or 1 resource can be fully covered with unused commitment discount costs. These latter two examples are commonly known as *commitment flexibility*.
 
-Since providers have different rules for when size-flexibility is or is not enabled, the following, fictitious SKU pricing attributes will be used in each example below, and each example will be categorized under size-flexibility enabled or disabled sections.
+When mixing usage-based commitment discounts with and without commitment flexibility with `CommitmentDiscountQuantity` measured by time, it is important to differentiate the `CommitmentDiscountUnit` for each type of commitment discount. In each scenario below, commitment discounts *without* commitment flexibility applied use "Hour" as a `CommitmentDiscountUnit`, and conversely commitment discounts *with* commitment flexibility use "Normalized Hour" as a `CommitmentDiscountUnit`.
```

Comment:
This paragraph was added to highlight the importance of using different units for commitment discounts with/without commitment flexibility